stdlib/hash.gr

```diff
@@ -247,7 +253,11 @@ let rec hashOne = (val, depth) => {
  */
```

Comment:
We should add the exception that `Result.unwrap` would throw to the grain doc to make sure it's known about.